### ISSUE TITLE
Windows_package resource is deprecated

### ISSUE THREAD

**Issue by spuder** [cvent/octopus-deploy-cookbook#29]
Don't use `windows_package` anymore since it is deprecated in favor of the built in `package` resource as of chef 12

        * octopus_deploy_tentacle[Tentacle] action install[2016-08-02T19:31:17+00:00] WARN: Please use the package resource available in Chef Client 12.6.
       windows_package will be removed in the next major version release
       of the Windows cookbook.

**Comment by spuder** [comment]
Nevermind, this shouldn't be an issue since the resource has the same name `windows_package`. Once the windows cookbook resource disappears, it should work as expected.

**Comment by brentm5** [comment]
At some point I will iron out what version of chef-client this cookbook will be supporting.  Right now we support chef-clients before 12.5.1 which is not ideal. This is why I explicitly use windows_package instead of package.  Just wanted to add this for further reference.

**Comment by spuder** [comment]
Now that the cookbook uses custom resources, it would be possible to revisit this issue. 

I see these deprecation warnings

```
       Recipe: nd-octopus::server
         * octopus_deploy_server[OctopusServer] action install[2017-09-20T22:30:12+00:00] WARN: Please use the package resource available in Chef Client 12.6.
       windows_package will be removed in the next major version release
       of the Windows cookbook.
```

**Comment by spuder** [comment]
To get rid of this warning, the windows cookbook version needs to be bumped. 
This is safe as long as users are running chef >12.5
